pkg/storage/db.go

```diff
@@ -61,8 +61,61 @@ const latestStatusQuery = "WITH latest_candidates AS ( " +
 	"    aws_resources ON " +
 	"        latest.aws_resources_id = aws_resources.id;"
 
+// Query to find resource by private IP using v2 schema
+// nolint
```

Comment:
question: what linting rules in particular does this break? (just curious)

Reply:
Unused constant.